ExampleFhirClient/Readme.md

```diff
@@ -0,0 +1,25 @@
+# ExampleFhirClient
+
+Very Basic Simple C# command line program to search, read and post information to/from the CareEvolution FHIR server.
+
+## Authentication
+   
+This application is using a JWT token to request an OAuth 2.0 access token and client authentication.  The application uses the jwt.careevolution.com certificate and expects the OAuth client configruation (id: JWTClientCredentials).  The signing certificate is available here: https://fhir.careevolution.com/jwt.pfx with password == `jwt.careevolution.com`
+
```

Comment:
Maybe add 'In a real production environment you would you your own signing certificate, providing CareEvolution its public key to configure the server'

Reply:
I've added 'in a real prod..' to help clarify.